lumicks/pylake/correlated_stack.py

```diff
@@ -14,13 +19,103 @@ class TiffFrame:
     page : tifffile.tifffile.TiffPage
         Tiff page recorded from a camera in Bluelake.
     """
-    def __init__(self, page):
+    def __init__(self, page, align):
         self._src = page
+        self._align = align
 
+    def _align_image(self):
+        """ reconstruct image using alignment matrices from Bluelake 
+            return aligned image as a NumPy array 
+        """
+
+        def correct_alignment_offset(alignment, x_offset, y_offset):
+            # translate the origin of the image so that it matches that of the original transform
+            transform1 = np.eye(3)
```

Comment:
I think it's pretty common knowledge, especially since it's also a Matlab function. It has more flexibility than identity because you can choose which diagonal you want filled, so it's my go-to